### PR TITLE
Show/hide other project layers on edit component #6623

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6623

** Front-end change only, test with Netlify

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectSummaryMap.js [hunk 1]
```diff
@@ -70,6 +71,8 @@ const ProjectSummaryMap = ({
    * Let's throw an error intentionally if there are no features for a project.
    */
   if (featureCount < 1) {
+    stopReportingRuntimeErrors();
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectSummaryMap.js:74]: i am so happy about this, i have been commenting out these lines when i develop locally.

File: moped-editor/src/views/projects/projectView/ProjectComponentEdit.js [hunk 2]
```diff
@@ -59,6 +59,7 @@ const useStyles = makeStyles(theme => ({
  * @param {function} handleCancelEdit - The function to call if we need to cancel editing
  * @param {function} projectRefetchFeatures - Reload parent component's features
  * @param {Object} projectFeatureCollection - The entire project's feature collection GeoJSON (optional)
+ * @param {function} projectSetFeatureCollection - Allows us to clear the feature collection for reloading
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectComponentEdit.js:62]: is this defined here in ProjectComponentEdit?

**Comment by amenity** [comment]
Adding @johnclary since he requested this. And I'm not _quite_ sure I understand the scope/ask.

File: moped-editor/src/views/projects/projectView/ProjectComponentEdit.js (hunk 2 above)
Comment (sergiogcx) [reply moped-editor/src/views/projects/projectView/ProjectComponentEdit.js]: @chiaberry 🙏 thank you so much, I totally missed that and the fact we do not directly use projectRefetchFeatures anymore. Just got that cleaned up.

**Comment by amenity** [comment]
🤦‍♀️ Looks like I never posted a comment I wrote yesterday on this PR. 

https://user-images.githubusercontent.com/1463708/126539525-26a174e0-bf0a-4db5-bcea-931ddb677a46.mov


tl;dr - 
- Can't click the aerial imagery toggle easily b/c it has the Mapbox logo/link on top of it
- I think the Nearmap token error is expected on Netlify; not sure if it should persist after the aerial layer is turned off?

**Comment by amenity** [comment]
This was incorporated into https://github.com/cityofaustin/atd-moped/pull/378
